libs/core/src/lib/checkbox/checkbox/checkbox.component.ts

```diff
@@ -61,6 +61,12 @@ export class CheckboxComponent implements ControlValueAccessor, AfterViewInit, O
     @ViewChild('labelElement')
     labelElement: ElementRef;
 
+    /** @hidden Whether input label should be wrapped */
+    @Input() wrapLabel: boolean;
+
+    /** @hidden Whether input label should be wrapped */
+    @Input() valignLabel: 'top' | 'middle' = 'middle';
```

Comment:
I am not sure about `valignLabel` name :D `alignLabel`?

Reply:
It's a reminiscent of valign attribute from table cells 😁 I think it describes purpose best 😁

Reply:
let's hope the user makes the same association :D

Reply:
Hope never dies